Cargo.toml

```diff
@@ -49,7 +49,7 @@ num-bigint = "0.4.6"
 pretty_assertions = "1.4.1"
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = { version = "1" }
-smol_str = { version = "0.2.2", features = ["serde"] }
+smol_str = { version = "0.3", features = ["serde"] }
```

Comment:
this is risky. I'd rather use the `smol_str` reexport from `cairo-lang-utils`

Reply:
Done.